ivy_tests/test_ivy/test_frontends/test_torch/test_blas_and_lapack_ops.py

```diff
@@ -225,6 +225,29 @@ def _get_dtype_input_and_vectors(draw, with_input=False, same_size=False):
     return dtype, vec1, vec2
 
 
+@st.composite
+def _get_dtype_lu_and_pivots(draw):
+    dimsize1 = draw(st.integers(min_value=2, max_value=5))
+    dimsize2 = draw(st.integers(min_value=2, max_value=5))
+    # dtype = draw(helpers.get_dtypes("valid"))
+    dtype = ["float32", "float64"]
+    # dtype = [
+    #     draw(st.sampled_from(tuple(set(dtype).difference({"bfloat16", "float16"}))))
+    # ]
+
+    mat = draw(
+        helpers.array_values(
+            dtype=dtype[0], shape=(dimsize1, dimsize2), min_value=1, max_value=10
+        )
+    )
```

Comment:
Why don't you use [`dtype_and_values`](https://unify.ai/docs/ivy/docs/helpers/ivy_tests.test_ivy.helpers.hypothesis_helpers/ivy_tests.test_ivy.helpers.hypothesis_helpers.array_helpers.html#ivy_tests.test_ivy.helpers.hypothesis_helpers.array_helpers.dtype_and_values)?